crates/driver/openapi.yml

```
@@ -430,6 +431,24 @@ components:
           description: The factor of the user surplus that the protocol will request from the solver after settling the order
           type: number
           example: 0.5
+    PriceImprovement:
+      description: |
+        A cut from the price improvement over the best quote is taken as a protocol fee.
+      type: object
+      properties:
+        kind:
+          type: string
+          enum: [ "priceImprovement" ]
+        maxVolumeFactor:
+          description: Never charge more than that percentage of the order volume.
+          type: number
+          example: 0.1
```

nit: I think our example value should not suggest to the reader that we might take 10% of their entire order.
```suggestion
          example: 0.01
```